### PR TITLE
Feature/co 1012

### PR DESCRIPTION
**Pull request by gmtcrary**
CO-1012

I refactored the mapping logic so all of the JSON handling fits in one screen.
I also added a docopt at the top so the command line arguments are easier to understand for a first time user.
Added a requirements.txt for the docopt import.

### CODE REVIEW

File: DataDiscrepancyUtil/dataDiffCheck.py [hunk 1]
```diff
@@ -1,61 +1,60 @@
...
-            old_bdata_name_dict[old_building['attributes']
-                                ['name']] = old_building['id']
+    
+    if(args['<old_data_path>'] == args['<new_data_path>']):
+        print "These are the same file..."
+        sys.exit(1)
+    else:
+
+        with open(args['<old_data_path>'], "r") as old_json_file:
+            with open(args['<new_data_path>'], "r") as new_json_file:
```
Comment (tsoliangwu0130) [inline DataDiscrepancyUtil/dataDiffCheck.py]: You could open multiple files like this:

```python
with open('a', 'w') as a, open('b', 'w') as b:
    do_something()
```

https://stackoverflow.com/questions/4617034/how-can-i-open-multiple-files-using-with-open-in-python

File: DataDiscrepancyUtil/dataDiffCheck.py [hunk 2]
```diff
@@ -1,61 +1,60 @@
...
+        new_data_path: File path of new building json
+"""
+# old.json being the data from https://api.oregonstate.edu/v1/locations?page[size]=10000&type=building
+# new.json being the data from https://localhost:8082/api/v0/locations?page[size]=10000&type=building - locations-frontend-api commit b1bec1e013cf29f16a0a01b9dd7c3777e3b4e192
+
+from docopt import docopt
+print(docopt(__doc__, version='1.0.0rc2'))
+
+args = docopt(__doc__, version='1.0.0rc2')
+
```
Comment (tsoliangwu0130) [inline DataDiscrepancyUtil/dataDiffCheck.py:17]: According to PEP8 standard, module level import should be at top of the file. See error [E402](http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes).

File: DataDiscrepancyUtil/dataDiffCheck.py [hunk 3]
```diff
@@ -80,14 +79,13 @@
                 if rekeyCheckDict.has_key(bname):
```
Comment (tsoliangwu0130) [inline DataDiscrepancyUtil/dataDiffCheck.py]: `has_key` has been deprecated, instead, we should use `in` here.

**Review by tsoliangwu0130 — CHANGES_REQUESTED** [review]
The code looks good, there are only a few [PEP8](https://www.python.org/dev/peps/pep-0008/) warnings. Could you fix these warning?

**Comment by gmtcrary** [comment]
Fixed, @tsoliangwu0130.

File: DataDiscrepancyUtil/dataDiffCheck.py (hunk 1 above)
Comment (gmtcrary) [reply DataDiscrepancyUtil/dataDiffCheck.py]: Thanks for showing me this!

**Review by tsoliangwu0130 — APPROVED** [review]
Looks good!

**Pull request merged**